src/_data/platforms.yml

```diff
@@ -1,50 +1,50 @@
 - platform: 'Android SDK'
   chipsets: 'x64, Arm32, Arm64'
   supported: '21 to 34'
-  besteffort: 'N/A'
+  ci-tested: '21 to 34'
   unsupported: '20 and earlier'
 - platform: 'iOS'
   chipsets: 'Arm64'
-  supported: '17'
-  besteffort: '12 to 16, 18'
+  supported: '12 to 18'
+  ci-tested: '16, 17'
```

Comment:
@jmagman do we have 16 on CI or only 17?

Reply:
Only 17.